src/modules/objects/controllers/objects.controller.ts

```diff
@@ -0,0 +1,70 @@
+import { Body, Controller, ForbiddenException, Headers, Logger, Param, Post } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { ApiParam, ApiTags } from '@nestjs/swagger';
+
+import { Configuration } from '~config';
+
+import { ObjectsQueryDto } from '../dto/objects.dto';
+import { ObjectMediaWithAggregations } from '../objects.types';
+import { ObjectsService } from '../services/objects.service';
+
+import { TranslationsService } from '~modules/translations/services/translations.service';
+import { SessionUserEntity } from '~modules/users/classes/session-user';
+import { Permission } from '~modules/users/types';
+import { RequireAllPermissions } from '~shared/decorators/require-permissions.decorator';
+import { SessionUser } from '~shared/decorators/user.decorator';
+
+@ApiTags('Objects')
+@Controller('objects')
```

Comment:
maybe also change this then to:
```suggestion
@Controller('ie-objects')
```

Reply:
Adjusted